### DISCUSSION TITLE
how to config my nginx to resolve the cors error?

### DISCUSSION THREAD

**Discussion by lixinchuang** [apache/skywalking#10942] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

NodeJS Client Side Agent (apache/skywalking-client-js)

### What happened

my nginx config like this :

   location /browser
	{
 
    add_header 'Access-Control-Allow-Origin' '*';
    add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS, PUT, DELETE';
     add_header 'Access-Control-Allow-Credentials' 'true';
    add_header 'Access-Control-Allow-Headers' 'Authorization,Content-Type,Accept,Origin,User-Agent,DNT,Cache-Control,X-Mx-ReqToken,X-Data-Type,X-Requested-With,X-Data-Type,X-Auth-Token';
 
		proxy_pass http://127.0.0.1:12800;
		#proxy_set_header x-forwarded-for  $remote_addr;
        proxy_set_header Host      $host;
        proxy_set_header X-Real-IP $remote_addr;
	}

but the cors errors are still there!

### What you expected to happen

how to config my nginx

### How to reproduce

what's wrong with the config?

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
client-js has nothing relative to your Ngnix, how to route is based on your deployment. I can't see the point of this question.

**Reply by lixinchuang** [reply]
my web port  and oap-server's port is not the same, so  when I  use client-js to report front information to the oap-server, it return CORS error. so I want  to use nginx to forward the request to the oap-server,  but the error did't disapear.  I just want to resolve the cors error.

**Reply by wu-sheng** [reply]
This is something you could learn by yourself. CORS isn't a new thing.

**Reply by tanshanshan** [reply]
maybe you could paste your code about visit nginx
